src/plugins/discover/public/application/components/sidebar/discover_field_details.tsx

```diff
@@ -76,13 +88,11 @@ export function DiscoverFieldDetails({
         </div>
       )}
 
-      {details.visualizeUrl && (
+      {showVisualizeLink && (
         <>
           <EuiLink
             onClick={() => {
-              getServices().core.application.navigateToApp(details.visualizeUrl.app, {
-                path: details.visualizeUrl.path,
-              });
+              triggerVisualizeActions(field, indexPattern.id, details.columns);
```

Comment:
Currently because we only have a `onClick` we can't use "Open in new tab" I think Anton mentioned, we could give the actions also an `getHref` and could potentially check here if there's only one action registered and if that has a `getHref` also assign it to the `EuiLink`s href, and thus enable Open in new Tab?

Reply:
getHref works when the popup appears, in other words when we have more that 1 action but we don't have here.
But I discussed it with @Dosant and he proposed a solution that I am going to introduce here

Reply:
I have added the getHref, unfortunately, the `<EuiLink` throws a linking error `<EuiLink> accepts either href or onClick, not both`. Waiting for Eui Team to answer me how we can handle this and remove the `eslint-disable-next-line`

Reply:
Well it seems that we will stick with the disabled rule for now https://github.com/elastic/eui/blob/336666aeff6edc3e730fea27751824d466004457/packages/eslint-plugin/index.js